manifests/plugin/gutterball.pp

```diff
@@ -13,5 +13,6 @@
     foreman_plugins_dir => $katello::config_dir,
     foreman_user        => $katello::user,
     foreman_group       => $katello::group,
+    notify              => [Service['foreman-tasks'], Service['foreman']],
```

Comment:
Often the foreman service is run through apache. It's be better to notify ```Class['foreman::service']``` which does the right thing for you.